setup.py

```diff
@@ -21,7 +21,7 @@ def read_file(fname):
     install_requires=[
         'pytest>=3.6',
         'requests>=2.20.0',
-        'simplejson',
+        'simplejson', 'freezegun', 'mock'
```

Comment:
didn't see those libs are used somewhere?

Reply:
Dear Petter,
I've just checked simplejson lib.
Seems like we don't use that in our project.
You can remove it.
Peace!!!